modules/agar/src/main/ts/ephox/agar/api/RealMouse.ts

```diff
@@ -58,6 +58,13 @@ const cClick = (): Chain<SugarElement<Element>, SugarElement<Element>> =>
 const pClickOn = (selector: string): Promise<{}> =>
   pActionOn(selector, 'click');
 
+const pClick = (elem: SugarElement<Element>): Promise<{}> => {
+  const id = Id.generate('');
+  Attribute.set(elem, BedrockIdAttribute, id);
+  const selector = `[${BedrockIdAttribute}="${id}"]`;
+  return pClickOn(selector);
```

Comment:
This doesn't remove the attribute once the action is done. We don't want to leave temporary state left around for this.

- [ ] Ensure the attribute is cleaned up after `pClickOn` completes